Resources/Prototypes/explosion.yml

```diff
@@ -58,6 +58,21 @@
   texturePath: /Textures/Effects/fire_greyscale.rsi
   fireStates: 3
 
+- type: explosion
+  id: Nuclear
+  damagePerIntensity:
+    types:
+      Heat: 5
+      Blunt: 5
+      Piercing: 5
+      Radiation: 5
+  tileBreakChance: [0, 0.5, 1]
+  tileBreakIntensity: [0, 10, 30]
+  tileBreakRerollReduction: 20
+  lightColor: Orange
+  texturePath: /Textures/Effects/fire.rsi
+  fireStates: 3
```

Comment:
Does this really need a new damage type? This adds overhead to the explosion tiles for something that is supposed to destroy everything in range anyway.